tf-modules/monitoring/dla.tf

```diff
@@ -44,27 +44,27 @@ resource "aws_glue_catalog_table" "dla_glue_table" {
     }
   
     columns {
-      name = "executionArn"
+      name = "executionarn"
```

Comment:
When we update these names and apply, is there any chance we end up with duplicate columns? If I understand the ticket correctly, `executionArn` will always end up as `executionarn` in Glue. Changing it to all lowercase shouldn't result in both `executionarn` and `executionArn` columns. Is that correct?

Also were you able to replicate the output ASF saw? e.g. when you initially picked this up, you saw the changeset mentioned in the ticket?

Reply:
From the ticket:
>but I've noticed that every time we deploy our terraform it seems to generate a plan to update the aws glue database column names

That's replicable without these changes?

Reply:
Ok, great. It seems like that validates the changes they were seeing and the fix removes the staged changes 👍